src/app/components/Table/index.tsx

```diff
@@ -166,6 +175,7 @@ export const Table: FC<TableProps> = ({
           <TablePagination {...pagination} />
         </Box>
       )}
+      {!isLoading && !rows?.length && emptyMessage && <CardEmptyState label={emptyMessage} />}
```

Comment:
This seems like it would render the `<table />` and then the empty message bellow it?

Reply:
The table header, and the empty message, yes. Like this:

![image](https://github.com/user-attachments/assets/47d0d8cb-ffc3-4e0d-881f-9f286b9494b5)

I find this more useful than hiding the whole table.

In any case, this was required in the initial version, when the filter widget for the TX type was placed in the table column header for type. In that arrangement, it was absolutely necessary that the table header stays, even if there is no data, so that we can change the filter to _get_ data. But we have changed the design, so now we could live with a disappearing table. Would you prefer that?

Reply:
Since there are no longer any filtering in the headers, so this is no longer necessary, so I have revised this decision. Now the table is fully hidden again.